library/core/src/ops/range.rs

```diff
@@ -62,12 +62,12 @@ impl fmt::Debug for RangeFull {
 /// assert_eq!(3 + 4 + 5, (3..6).sum());
 ///
 /// let arr = [0, 1, 2, 3, 4];
-/// assert_eq!(arr[ ..  ], [0,1,2,3,4]);
-/// assert_eq!(arr[ .. 3], [0,1,2    ]);
-/// assert_eq!(arr[ ..=3], [0,1,2,3  ]);
-/// assert_eq!(arr[1..  ], [  1,2,3,4]);
-/// assert_eq!(arr[1.. 3], [  1,2    ]);  // Range
```

Comment:
I'm not sure why the comment `// Range` was here; it didn't make sense to me, so I removed it.

Reply:
It looks like these must have been copy-pasted to all the `Range*` types?  So the comment is probably to emphasize that that particular line is the one for _this_ type.

Seems like this example should be maybe somewhere where we reference it from all the types, and call-out `Range`, `RangeTo`, etc on every line.  I guess there's not a `core::range` module to put it on...

Maybe this example could go on `RangeBounds`, and all the `Range*` types could link over there as a way to jump between them?

Reply:
> So the comment is probably to emphasize that that particular line is the one for _this_ type.

Ah, I see; thanks for clearing that up :)

Reply:
For now I just updated all the examples to make them consistent.